distribution/docs/src/main/resources/_contents/_endpoints/cometd-endpoint-contents.adoc

```diff
@@ -0,0 +1,756 @@
+|Any JSON Type
+|Any JSON
+|"{}"
+
+|===
+
+.Example: Activity update with custom 'bytes' field
+[source,json,linenums]
+----
+data: {
+  "category": "Product Retrieval",
+  "event.topics": "ddf/activities/broadcast",
+  "id": "a62f6666-fc41-4a19-91f1-485e73a564b5",
+  "message": "The requested product is being retrieved. Standby.",
+  "operations": {
+    "cancel" : true
+  },
+  "progress": "",
+  "status": "RUNNING",
+  "timestamp": "1403801920875",
+  "title": "Product downloading",
+  "user": "admin",
+  "bytes": 635084800
+}
+----
+
+====== Activity Operations Channel
+
+Different operations can be performed on activities through the `/service/action` channel.
+
+.CometD Activity Format
+[cols="1m,2,1,1,2,2" options="header"]
+|===
+|Map Key
+|Description
+|Value Type
+|Possible Values
+|Example Values
+
+|action
+|The requested action. This value is based on the operations map that comes in from an activity event.
+|String
+|Any String
+a|Common values are
+
+* "download" +
+* "retry" +
+* "cancel" +
+* "pause" +
+* "remove" +
+* "resume" +
+* "cancel"
+
+|id
+|ID of the activity to which the requested operation relates
+|String
+|Any String
+|"a62f6666-fc41-4a19-91f1-485e73a564b5"
+
+|===
+
+.Example: Activity Operation Request Message
+[source,json,linenums]
+----
+"data": [ {
+	"action":"cancel",
+ 	"id":"a62f6666-fc41-4a19-91f1-485e73a564b5"
+} ]
+----
+
+===== Query Request Examples
+
+.Enterprise Contextual Query
+[source,json,linenums]
+----
+"data": {
+	"count": 250,
+ 	"format": "geojson",
+ 	"id": "4303ba5d-21af-4878-9a4c-808e80052e6c",
+ 	"cql": "anyText LIKE '*'",
+ 	"src": "${branding}-OS,${branding-lowercase}.distribution",
+ 	"start": 1
+}
+----
+
+.Multiple Site Temporal Absolute Query
+[source,json,linenums]
+----
+"data": {
+	"count": 250,
+	"format": "geojson",
+	"id": "4303ba5d-21af-4878-9a4c-808e80052e6c",
+	"cql": "modified DURING 2014-09-01T00:00:00Z/2014-09-30T00:00:00Z",
+ 	"src": "${branding}-OS,${branding-lowercase}.distribution",
+ 	"start": 1,
+}
+----
+
+.Enterprise Spatial Bounding Box Query
+[source,json,linenums]
+----
+"data": {
+	"count": 250,
+	"format": "geojson",
+	"id": "4303ba5d-21af-4878-9a4c-808e80052e6c",
+	"cql": "INTERSECTS(anyGeo, POLYGON ((-112.7786 32.2159, -112.7786 45.6441, -83.7297 45.6441, -83.7297 32.2159, -112.7786 32.2159)))",
+	"start": 1
+}
+----
+
+====== Query Response Channel
+
+The query responses are returned on the `/<id>` channel, which should be subscribed to in order to retrieve the results.
+Replace `<id>` with the id that was used in the request.
+The <<_asynchronous_search_subscribing, Subscribing to Notifications>> section details how to subscribe to a CometD channel.
+
+The response is returned as a data map that contains an internal map with the following keys:
+
+.Query Response Message Format
+[cols="1m,3,1,1,1" options="header"]
+|===
+|Map Key
+|Description
+|Value Type
+|Possible Values
+|Example Values
```

Comment:
Wonder if we shouldn't get rid of the `example` column and put any useful information it contains in the `Description`.